packages/graphql-yoga/src/plugins/resultProcessor/push.ts

```diff
@@ -18,7 +18,7 @@ export function processPushResult(
 
   const responseInit = getResponseInitByRespectingErrors(result, headersInit)
 
-  let iterator: AsyncIterator<ExecutionResult<any>>
+  let iterator: AsyncIterator<ExecutionResult<any> | ExecutionResult<any>[]>
```

Comment:
We can use the MaybeArray utility here too?

Reply:
Done!